modules/ossm-vs-istio.adoc

```diff
@@ -10,27 +10,44 @@ An installation of {ProductName} differs from upstream Istio community installat
 The current release of {ProductName} differs from the current upstream Istio community release in the following ways:
 
 [id="ossm-multi-tenant-install_{context}"]
-== Multi-tenant installations
+== {ProductName} control plane
 
+{ProductName} installs a multi-tenant control plane by default. You specify the projects that can access the {ProductShortName}, and isolate the {ProductShortName} from other control plane instances.
+
+[id="ossm-mt-vs-clusterwide_{context}"]
+=== Multi-tenancy in {ProductName} versus cluster-wide installations
+
+The main difference between a multi-tenant installation and a cluster-wide installation is the scope of privileges used by the control plane deployments, for example, Galley and Pilot. The components no longer use cluster-scoped Role Based Access Control (RBAC) `ClusterRoleBinding`, but rely on {ProductShortName} project-scoped RBAC `ServiceMeshRbacConfig`.
```

Comment:
Change last sentence to:
"The components no longer use cluster-scoped Role Based Access Control (RBAC) resource `ClusterRoleBinding`, but rely on project-scoped `RoleBinding`."

ServiceMeshRbacConfig is something completely different. :)

Reply:
Fixed.